RapidReact/src/main/java/frc/robot/Robot.java

```diff
@@ -340,15 +330,24 @@ private void configureButtonBindings() {
       buttons.lowShoot.whenPressed(shooterSubsystem::shootLow);
       buttons.lowShoot.whenReleased(shooterSubsystem::stopShoot);
 
-      buttons.hubShoot.whenPressed(() -> {
-        shooterSubsystem.shootTop();
+      buttons.hubSpinUp.whenPressed(() -> {
+        shooterSubsystem.spinUpTop();
       });
-      buttons.hubShoot.whenReleased(() -> {
+      buttons.hubSpinUp.whenReleased(() -> {
         shooterSubsystem.stopShoot();
         if (config.enableIntakeSubsystem) {
           intakeSubsystem.stopBallManagement();
         }
       });
+
+      buttons.feedInFire.whenPressed(() -> {
```

Comment:
Don't you only want this button to do anything when the shooter is up to speed?

Reply:
no, this is operator controlled firing.

Reply:
That seems really sketchy. In what scenario would you want to let the operator fire the balls when the shooter isn't up to speed?

Reply:
When we refine the PID control of the shooter, we can turn back on upToSpeed functionality. Right now, it only takes a second to spin up, so as long as the driver isn't too trigger happy, it'll be fine.

With our current PID, the isUpToSpeed is not reliable enough. Sometimes it works, sometimes it doesn't get close enough to our error params. This means the driver would be trying to fire and the motors wouldn't turn on because the speed wasn't quite in our error range.